app/serializers/web_hook_post_serializer.rb

```diff
@@ -0,0 +1,200 @@
+class WebHookPostSerializer < ApplicationSerializer
```

Comment:
For topic, it doesn't serialize post stream and ~10 attributes. For posts and users, ~10 less at least.